packages/hotspot/configure/src/DeleteWidget.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Group } from 'react-konva';
+import { faDelete } from './icons';
+import Image from './image';
+
+function calculate(polygonPoints) {
```

Comment:
Can we please add sanity checks? Also, this new file could use some tests, right?